### PR TITLE
Winograd wsc

### PR DESCRIPTION
**Pull request by arunraja-hub**

### CODE REVIEW

File: templates/commonsense_qa/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+      {{ choices[''text''][choices[''label''].index(answerKey)] }}'
+    name: most_suitable
+    reference: ''
+  bc718994-1d3e-4ae4-b65b-be307154b0a6: !Template
+    id: bc718994-1d3e-4ae4-b65b-be307154b0a6
+    jinja: 'Use the following options to predict a possible question for them:
+
+
+      {% for i in range(choices[''text'']|length) %}
```
Comment (srush) [inline templates/commonsense_qa/templates.yaml:53]: You should use {{choices["text"] | join("...")}} for this.

File: templates/winograd_wsc/wsc273/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    id: c07d5f26-303f-44d9-a941-659debdd9ec2
+    jinja: 'Who does the pronoun "{{pronoun}}" in "{{text}}" refer to? |||
+
+      {{options[label]}}'
+    name: pronoun
+    reference: ''
+  df01735f-5f06-408f-930d-618f3cfc9261: !Template
+    id: df01735f-5f06-408f-930d-618f3cfc9261
+    jinja: "Who does the pronoun \"{{pronoun}}\" in \"{{text}}\" refer to?\n\nThe\
+      \ options are:\n{% for i in range(options|length) %}\n- {{options[i]}}\n{% endfor\
```
Comment (srush) [inline templates/winograd_wsc/wsc273/templates.yaml:25]: Join

**Review by srush — CHANGES_REQUESTED** [review]
Small changes

**Comment by arunraja-hub** [comment]
@srush I accidentally added the commonsense_qa file in this PR for which you have added the comment. Now I have removed it. Could you help to review this again please?

**Comment by srush** [comment]
I don't think my comments above were addressed yet? Use join instead of for.

**Pull request merged**